tests/test_render.py

```diff
@@ -14,13 +14,22 @@
 
 
 @pytest.mark.parametrize(
-    "svg_file",
+    ("svg_file", "options"),
     [
-        pytest.param("resources/examples/svg/octocat.svg", id="octocat"),
+        pytest.param(
+            "resources/examples/svg/octocat.svg",
+            resvg_py.SVGOptions(
+                shape_rendering=resvg_py.ShapeRendering.GeometricPrecision,
```

Comment:
@jboarman Do you have handy examples of input/output pairs that might be worth testing here?

There's [a bunch of examples](https://github.com/RazrFalcon/resvg/tree/master/tests/svg) in the upstream Rust repo, but it's probably overkill to check all of them.

Reply:
Until we implement the benchmark in issue #10, I think we can make use of a few random selections from the upstream repo that you identified in your comment.

Reply:
Alright, I'll try to port a few of the test files to validate the options.